x/dispensation/types/records.go

```diff
@@ -28,8 +38,15 @@ func (dr DistributionRecord) Add(dr2 DistributionRecord) DistributionRecord {
 	return dr
 }
 
+func (dr DistributionRecord) DoesClaimExist() bool {
```

Comment:
This function name doesn't seem to indicate what it's doing, it's not checking if a claim exists in the store.

Reply:
Its called from distribute drops .  The although doesnt query the keeper , it certifies that the claim should exist . 
Alternatively I can rename this to something like DoesTypeSupportClaim()